README.md

```diff
@@ -1,13 +1,13 @@
 # GitHub Tracker
 
-Replace this readme with your own information about your project.
-
-Start by briefly describing the assignment in a sentence or two. Keep it short and to the point.
+The assignment was to create a place to keep track of the GitHub repos that I use at Technigo. The focus was JavaScript and API.
 
 ## The problem
 
-Describe how you approached to problem, and what tools and techniques you used to solve it. How did you plan? What technologies did you use? If you had more time, what would be next?
+In the beginning of the project it felt like there were a lot of API fetches that needed to be done and. It was also hard to understand were to place, and where to call the different functions. I also struggled with writing short and clean code and sometimes ended up with writing several codes extra.
+
+The solution were to scale down the project and do small chunks at a time. Create a TODO and check off everything during the work progress.
```

Comment:
This is a great idea. Maybe including the To Do list somewhere in the README (or a separate text file) could be even more illustrative for those of us who are curious.